github/github.go

```diff
@@ -357,6 +357,14 @@ func (c *Client) Do(req *http.Request, v interface{}) (*Response, error) {
 	return response, err
 }
 
+// Block represents more detailed error information such as is returned
+// for a code 451. See https://developer.github.com/changes/2016-03-17-the-451-status-code-is-now-supported/
+// for more information.
+type Block struct {
```

Comment:
I worry about created a new top-level, exported type just for this very obscure kind of error.  How bout declaring this as an anonymous inner struct on the ErrorResponse.  Something like http://play.golang.org/p/tmW_cl2pGr